airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionNotificationWorkflowImpl.java

```diff
@@ -35,14 +34,14 @@ public class ConnectionNotificationWorkflowImpl implements ConnectionNotificatio
   @Override
   public boolean sendSchemaChangeNotification(final UUID connectionId)
       throws IOException, InterruptedException, ApiException, ConfigNotFoundException, JsonValidationException {
-    final StandardSync standardSync = configFetchActivity.getStandardSync(connectionId);
+    final Optional<Boolean> breakingChange = configFetchActivity.getBreakingChange(connectionId);
```

Comment:
I think that there is a (small) risk to have workflow being stuck because of a lack of version. could we add one in order to avoid any issue.